sw/blas/gemm/src/gemm_fp32.h

```diff
@@ -53,17 +58,22 @@ void gemm_fp32_naive(uint32_t M, uint32_t N, uint32_t K, float* A, uint32_t ldA,
     }
 }
 
-void gemm_fp32_naive_unrolled(uint32_t M, uint32_t N, uint32_t K, float* A,
-                              uint32_t ldA, uint32_t ta, float* B, uint32_t ldB,
-                              uint32_t tb, float* C, uint32_t ldC, float BETA) {
+void gemm_fp32_naive_unrolled(uint32_t M, uint32_t N, uint32_t K, void* A_p,
+                              uint32_t ldA, uint32_t ta, void* B_p,
+                              uint32_t ldB, uint32_t tb, void* C_p,
+                              uint32_t ldC, uint32_t BETA, uint32_t setup_SSR) {
```

Comment:
I'm a bit confused cause here we are changing beta from floating-point to integer, but the PR describes the opposite change. Could you explain?

Reply:
You are right, I removed the comment. This will be fixed in a separate PR, since we have to account for multiplicative factors `beta` other than 0 or 1.